test/integration/testpaths/testpath_snapshot_limits.py

```diff
@@ -358,3 +362,129 @@ def test_01_storage_snapshots_limits(self):
         )
 
         return
+
+    @attr(tags=["advanced", "basic"], required_hardware="true")
+    def test_02_snapshot_size_check(self):
+        """ Check Snapshots size in database
+            1. Create file on ROOT disk of deployed VM.
+            2. Create Snapshot of ROOT disk.
+            3. Check if physiacal_size parameter of snapshot_store_ref table
+               has physical size of snapshot
+        """
+        if self.hypervisor.lower() not in ["xenserver", "vmware"]:
+            self.skipTest("Test not to be run on %s" % self.hypervisor)
```

Comment:
Here we are running the test for Xenserver and VMWare  i.e. skip the test if the hypervisor is not XS or VMWare .
So the test run on Xenserver.